### DISCUSSION TITLE
part-ii/2021-121/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#369] [Question comments]
Part II 2021 Probability and Measure: Paper 1, Section II, H 

https://questions.tripos.org/part-ii/2021-121/

**Comment by ArthurConmy** [comment]
<details>
<summary>b)</summary>
E.g $\mathbb{ R}$ with Lebesgue measure and $f_n = 1_{ [n,n+1] }$.
</details>

<details>
<summary>c)</summary>
Yes, Fatou's gives $\mathbb{ E}[X] \le \text{liminf} \mathbb{ E}[X_n] \le \text{sup}_n \mathbb{ E}[X_n]$.

[feels so quick was something missed?!]
</details>
